### DISCUSSION TITLE
How to change the font preview to simplified characters

### DISCUSSION THREAD

**Discussion by zhenzi0322** [fontforge/fontforge#5175] [Q&A]
![image](https://user-images.githubusercontent.com/34839719/211028167-7b803233-b5b3-4a53-9d24-556df41b04bc.png)

**Comment by ctrlcctrlv** [comment]
Good question. I honestly don't know. It's got to be some font metadata thing.

I only use free software so I can't try different things, but here's some things to try:

1. Mark the font as supporting MS Code Page 936 in OS/2 table.
    ![image](https://user-images.githubusercontent.com/838783/211059593-f86768cd-3e85-4132-8df3-acfc650154ec.png)
2. Make sure you only have English and Chinese (PRC) strings in `name` table.
    ![image](https://user-images.githubusercontent.com/838783/211059970-0500d762-dd43-46b3-b1d9-110a66237f87.png)
3. Make sure LanguageGroup is 1 in PS Private. 
    ![image](https://user-images.githubusercontent.com/838783/211060187-7a1aa26a-4dc9-45a8-b842-200dfedc19de.png)

Good luck.
